msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/SignInStates.kt

```diff
@@ -244,26 +245,17 @@ class SignInCodeRequiredState internal constructor(
                     )
                 }
 
-                is INativeAuthCommandResult.Redirect -> {
-                    ResendCodeError(
-                        errorType = ErrorTypes.BROWSER_REQUIRED,
-                        error = result.error,
-                        errorMessage = result.errorDescription,
-                        correlationId = result.correlationId
-                    )
-                }
-
-                is INativeAuthCommandResult.UnknownError -> {
+                is INativeAuthCommandResult.Redirect, is INativeAuthCommandResult.UnknownError -> {
```

Comment:
For the sake of clarity, I would prefer to have line breaks between these results.

Reply:
I don't agree with that. It's not a Kotlin code style I've seen before.